concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1440,14 +1479,14 @@ jobs:
     plan:
       - aggregate:
           - get: pipeline-trigger
-            passed: [ 'cf-deploy']
+            passed: ['post-deploy','availability-tests']
```

Comment:
`avail-test` will tell you if app is reachable again. It's not failing immediately but measuring error rate and providing statistics at the end. I see no point in continuing running the pipeline after failure. If you want to run other tests, run them manually.